### DISCUSSION TITLE
"new" data layers to incorporate to Eurocrops?

### DISCUSSION THREAD

**Discussion by caropeich** [maja601/EuroCrops#12] [Q&A]
Hi, Firs big thanks to all the contributors to this high value project.

This short message just to send the link to LPIS layers, from 2017-Present. from Andalucia (Spain). https://www.juntadeandalucia.es/organismos/agriculturaganaderiapescaydesarrollosostenible/servicios/sigpac/visor/paginas/sigpac-descarga-informacion-geografica-shapes-provincias.html.

Also this one can be of interest, including also other areas in Spain with open layers. https://mappinggis.com/2020/03/como-descargar-las-parcelas-del-sigpac-en-qgis-el-caso-de-castilla-y-leon/

Open to help to include them in EUROCROP if you find it useful,

**Comment by maja601** [comment]
Hi @caropeich,
thank you for your kind words and the link for Andalucia!
Adding Spain to the dataset is unfortunately at the moment quite difficult due to the regional divisions. As most of the work of the dataset is still done by hand, I don't have the resources to fit them all in manually. But in the future, we are hoping to add Spain as an entire layer.
If you are interested in contributing, it is always possible to create a mapping csv file (just like the ones in [here](https://github.com/maja601/EuroCrops/tree/main/csvs/country_mappings) with the following properties yourself:
| Original Code | Original Name | Translated Name | HCAT name | HCAT ID |
| ------------- | --------------- | ---------------- | ------------ | --------- |

for the dataset you are interested in and perform a join on the original data e.g. the shapfile from Andalucia with the mapping file over the Original Code as a key.
We would then add this to EuroCrops and your name to the contributor list.

Cheers,
Maja

**Reply by caropeich** [reply]
thanks a lot @maja601 for the super-fast reply. I'll do my best to complete the csv file and send it back. Cannot promise to be fast, but I will, for sure. As the codes are pretty similar to the rest of Spain we can hopefuly then complete it faster for the rest. :)

**Reply by maja601** [reply]
That would be fantastic @caropeich, we are currently also looking at a new data sources, as the one we had for e.g. Navarra had really coarse (more general Land Cover, less the precise crop type) classes. I don't know how granular the data is that you are currently looking at, but maybe have an eye on that. And I can let you know, what source we are using if we find a good one.

**Reply by caropeich** [reply]
yes, I am aware of challenging this case is. keep in touch

**Comment by caropeich** [comment]
please @maja601 take a look to this file for Andalucia and see if that looks fine. At the end, pretty similar to the one already there for Navarra. So I guess most of the codes will be shared for the whole country. However, I found some issues that maybe worth a discussion for homogeinizing it. for example: for Navarra I saw 
TH	HUERTA	VEGETABLE PLOT	fresh_vegetables	3301070000

While I decided to leave it as:

TH	HUERTA	ORCHARD	arable_crops	3301000000

Some other minor discrepancies as this one are there. Mostly are due to translation and local knowledge of the terms, I think.

- Apart from that, I found some others (now on not_known_and_other	3399000000) that maybe worth to be added as new categories in the HCAT2 matrix


[UsosSIGPAC_HCAT2_ES_Andalucia.csv](https://github.com/maja601/EuroCrops/files/9584433/UsosSIGPAC_HCAT2_ES_Andalucia.csv)

**Reply by maja601** [reply]
Hi @caropeich thank you so much for putting the mapping together; It looks good!
As you said, I also think it makes sense to discuss these things, maybe over zoom? Could you please send me an E-Mail, then we could set a date & time?
You can find it [here](https://www.asg.ed.tum.de/en/lmf/schneider/) :)
